api/app/invitedGuest/tests.py

```diff
@@ -133,3 +133,126 @@ def test_create_invitedGuest_list(self):
                          INVITED_GUEST['email'])
         self.assertEqual(data[1]['user']['email'],
                          INVITED_GUEST_2['email'])
+
+class InvitedGuestTagAPITest(ApiTestCase):
+    def _seed_static_data(self):
+        self.event1 = self.add_event(key='event1')
+        self.event1admin = self.add_user('event1admin@mail.com')
+        self.event1reviewer1 = self.add_user('event1reviewer1@mail.com')
+        self.event1reviewer2 = self.add_user('event1reviewer2@mail.com')
+        self.user1 = self.add_user('user1@mail.com')
+        self.user2 = self.add_user('user2@mail.com')
+        self.invited_guest1 = self.add_invited_guest(user_id=self.user1.id, event_id=self.event1.id, role="Guest")
+        self.invited_guest2 = self.add_invited_guest(user_id=self.user2.id, event_id=self.event1.id, role="Mentor")
+
+        self.event1.add_event_role('admin', self.event1admin.id)
+        self.event1.add_event_role('reviewer', self.event1reviewer1.id)
+        self.event1.add_event_role('reviewer', self.event1reviewer2.id)
+
+        self.tag1 = self.add_tag()
+        self.tag2 = self.add_tag(names={'en': 'Tag 2 en', 'fr': 'Tag 2 fr'})
+
+        self.tag_invited_guest(self.invited_guest1.id, self.tag1.id)
+        self.tag_invited_guest(self.invited_guest2.id, self.tag2.id)
+
+    def test_tag_admin(self):
+        """Test that an event admin can add a tag to an invited guest."""
+        self._seed_static_data()
+
+        params = {
+            'event_id': self.event1.id,
+            'tag_id': self.tag1.id,
+            'invited_guest_id': self.invited_guest1.id
+        }
+
+        response = self.app.post(
+            '/api/v1/invitedguesttag',
+            headers=self.get_auth_header_for('event1admin@mail.com'),
+            json=params)
+
+        self.assertEqual(response.status_code, 201)
+
+        params = {
+            'event_id': self.event1.id,
+            'invited_guest_id' : self.invited_guest1.id,
+            'language': 'en',
+            'include_unsubmitted': False
+        }
+
+        response = self.app.get(
```

Comment:
This get call is causing the test to fail. The error message is : {'message': {'email': 'Missing required parameter in the JSON body or the post body or the query string'}}. Any ideas?

Reply:
Yeah Get requests don't seem to work with the json property for params (not entirely sure why), so we usually do it in the query string like this: https://github.com/deep-learning-indaba/Baobab/blob/develop/api/app/invitedGuest/tests.py#L127